pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -243,7 +243,7 @@ public static String rtrim(String input) {
    * @param input
    * @return get substring starting from the first index and extending upto specified length.
    */
-  @ScalarFunction
+  @ScalarFunction(names = {"leftSubStr", "left"})
```

Comment:
Is `leftSubStr` a valid function in any common DB? Should we add the underscore version?

Reply:
I had checked, with v1, all variations are supported `left_substr`, `leftSubStr`, `left_sub_str`, which is why I didn't add them explicitly. For v2, we want to be Postgres compliant, so there is just `left`.